scripts/lib/installer-win32-gen.tsx

```diff
@@ -163,6 +164,7 @@ export default async function generateFileList(rootPath: string): Promise<string
   const descendantDirs = getDescendantDirs(rootDir).filter(d => d.files.length > 0);
 
   const specialComponents: Record<string, (d: directory, f: { name: string, id: string }) => Element | null> = {
+    // @ts-ignore
```

Comment:
Do we have a follow-up to fix this? The commit message in cee498a3a63be077af4c9e4f7983bdcf9b46d63b says we plan to use `JSX.Element` directly, but that doesn't appear to be implemented.

Reply:
I'm thinking that we might have lost some context after squashing. I'll need to clean up this commit message as we don't utilize `JSX.Element` anymore. 

I created #5543 to address